bin/check-dns.rb

```diff
@@ -96,15 +96,39 @@ class DNS < Sensu::Plugin::Check::CLI
          long: '--use-tcp',
          boolean: true
 
+  option :count,
```

Comment:
I thought about this more and I can not justify any reason to not make this more descriptive. Change this and we can merge this.